exercises/concept/authentication-system/AuthenticationSystemTests.cs

```diff
@@ -14,6 +14,17 @@ public void GetAdmin()
         Assert.Equal(admin, authenticator.Admin);
     }
 
+    [Fact]
+    [Task(4)]
+    public void CheckAdminCannotBeTampered()
+    {
+        var admin = new Identity { EyeColor = "green", Email = "admin@ex.ism" };
+        var authenticator = new Authenticator(admin);
+        var tamperedAdmin = authenticator.Admin;
+        tamperedAdmin.Email = "admin@hack.ed";
+        Assert.NotEqual(tamperedAdmin.Email, authenticator.Admin.Email);
```

Comment:
Maybe do:

```suggestion
        Assert.Equal("admin@ex.ism", authenticator.Admin.Email);
```

This is a slightly stronger assertion, as the current test theoretically still passes when the admin email is changed to something else than the tampered admin.

Reply:
Totally agree!

Reply:
Changes done.